src/ch/blinkenlights/android/vanilla/RemoteControlImplLp.java

```diff
@@ -111,6 +116,17 @@ public void reloadPreference() {
 		mShowCover = -1;
 	}
 
+	// HACK: This static is only needed as I found no other way to force correct progress display in the following scenarios:
+	// I.  1. Vanilla is started after a "Force stop"
+	//     2. BlueSoleil 10.0.497.0 is connected
```

Comment:
What is BlueSoleil and why do we need to implement a workaround for it?
Also, these changes would probably be better off in a separate pull request.

Reply:
OK, I'll split this PR into separate ones.

Reply:
BlueSoleil an AVRCP-capable Bluetooth receiver for Windows (http://www.bluesoleil.com/products/S0001201005190001.html).
I think it makes perfect sense to make sure that Vanilla's interoperation with the Remote Control code is done in such a way that as many publicly available receivers as possible work as perfectly as possible, so BlueSoleil is one good test case for this goal. It is not clear which of the components in the pair "Remote Control code in Android + BlueSoleil AVRCP code" is guilty of the scenario I described in the comments for which this workaround is intended, and it is possible that one of those components makes some extra assumptions not guaranteed by the AVRCP specifications, but it is not realistic to expect one of them to fix this, so implementing this (fairly benign and harmless) workaround is no doubt a reasonable thing for us to do.